strawberry/aiohttp/views.py

```diff
@@ -127,57 +127,62 @@ async def handle_connection_terminate(self, ws: web.WebSocketResponse) -> None:
         await ws.close()
 
     async def handle_start(
-        self, request: web.Request, ws: web.WebSocketResponse, message: dict
+        self, request: web.Request, ws: web.WebSocketResponse, message: OperationMessage
     ) -> None:
         operation_id = message["id"]
-        payload: dict = message["payload"]
+        payload: StartPayload = message["payload"]  # type: ignore
```

Comment:
@patrick91 mypy didn't know which type from the `payload` `Union` to expect here so I added `payload: StartPayload`. Then mypy complained again:

```
error: Incompatible types in assignment (expression has type "Union[Dict[str, Any], StartPayload, DataPayload, ErrorPayload]", variable has type "StartPayload")
```
To deal with this I added `# type: ignore`. Do you know an alternative solution?

Reply:
yes! we can use type guards in future, but for now I'd use `cast(StartPayload, message["payload"])` :)

Reply:
we can update to use cast or merge now and update to use typeguards in future, I'll let you decide :D

Reply:
I just tried using TypeGuards (interesting concept), unfortunately our mypy version doesn't play nice with them: `error: Variable "typing_extensions.TypeGuard" is not valid as a type`. Temporarely updating mypy solved this issue, but mypy then complaints about veriours other things (like library stubs not being installed for flask, dateutil and click). I'll probably use TypeGuards and somehow satisfy mypy (probably using type ignore). When we update mypy the type ignore can be removed.

Reply:
Forget the last part, I went with `cast` for now. Using TypeGuards became a little tricky since a functions input is validated to have a certain type but I need the whole message to confirm its payload has a certain type. I'll look at typeguards in more depth in a separate PR.